src/org/zaproxy/zap/extension/pscanrulesAlpha/ZapAddOn.xml

```diff
@@ -9,6 +9,8 @@
 	<![CDATA[
 		Issue 2538: Strict-Transport-Security Header Scanner additional variants.<br>
 		Issue 2716: Implement a passive check for cookies without SameSite set.<br>
```

Comment:
11 was in fact released (https://github.com/zaproxy/zap-extensions/releases) so the previous change entries can be removed.